setup.py

```diff
@@ -4,10 +4,19 @@
     long_description = f.read()
 
 with open("requirements.txt") as f:
-    requires = f.read().splitlines()
+    requires = []
+    for line in f:
+        req = line.strip()
+        if "#egg=" in req:
+            req_url, req_name = req.split("#egg=")
+            req_str = f"{req_name} @ {req_url}"
+        else:
+            req_str = req
+        print(req_str)
+        requires.append(req_str)
 
 setuptools.setup(
-    name="ondewo-nlu-client-python",
+    name="ondewo-nlu-client",
```

Comment:
waaa, didnt i do that already? or did noone merge my PR

Reply:
I noticed the pypi version is called `ondewo-nlu-client`, but haven't seen it in history of this repo :thinking: